### ISSUE TITLE
Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by Smerdokryl** [WowDevTools/CASCHost#14]
CASCHost was working more than fine, but then I stopped it to clear the output folder, and after that it refused to run again (including after cloning the repo and building it again).

I'm getting the following errors:
```
info: CASCHost.Startup[0]
      Loading cache...

Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at CASCEdit.Helper.GetCDNPath(String filename, String type, String basepath) in C:\Games\Server\CASCHost\CASCEdit\Helper.cs:line 9
   at CASCHost.Cache.ReadAll(DbDataReader reader) in C:\Games\Server\CASCHost\CASCHost\Cache.cs:line 183
   at CASCHost.Cache.LoadOrCreate() in C:\Games\Server\CASCHost\CASCHost\Cache.cs:line 136
   at CASCHost.Startup.Configure(IApplicationBuilder app, IHostingEnvironment env, IServiceProvider serviceProvider, IOptions`1 settings) in C:\Games\Server\CASCHost\CASCHost\Startup.cs:line 63
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Hosting.ConventionBasedStartup.Configure(IApplicationBuilder app)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.BuildApplication()
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.Build()
   at CASCHost.Program.Main(String[] args) in C:\Games\Server\CASCHost\CASCHost\Program.cs:line 24
```
![image](https://user-images.githubusercontent.com/35412834/41089765-875242dc-6a3a-11e8-8ffc-db900e8a625d.png)
![image](https://user-images.githubusercontent.com/35412834/41089772-8a8aa49e-6a3a-11e8-99b0-f6911ca12afd.png)

**Comment by Smerdokryl** [comment]
Deleting the whole root_entries table seems to have fixed it, even though it is not a recommended action afaik

**Comment by barncastle** [comment]
Should be fixed via 7146a404675315bcc46ab1e289d7ad6d1cb642ed
